src/Explorer/Notebook/NotebookContentClient.ts

```diff
@@ -117,8 +117,11 @@ export class NotebookContentClient {
 
   private async checkIfFilepathExists(filepath: string): Promise<boolean> {
     const parentDirPath = NotebookUtil.getParentPath(filepath);
-    const items = await this.fetchNotebookFiles(parentDirPath);
-    return items.some(value => FileSystemUtil.isPathEqual(value.path, filepath));
+    if (parentDirPath) {
+      const items = await this.fetchNotebookFiles(parentDirPath);
+      return items.some(value => FileSystemUtil.isPathEqual(value.path, filepath));
+    }
+    return undefined as any;
```

Comment:
we should return either false or throw an error

Reply:
Sure, I will changed that